### ISSUE TITLE
Multiple Language support required

### ISSUE THREAD

**Issue by Agrover112** [Agrover112/awesome-semantic-search#31]
**Is your feature request related to a problem? Please describe.**
Support for multiple languages is required keeping with the essence of FOSS.

**Describe the solution you'd like**
Contributors who can understand English well enough and also native language to translate contents of the repository.

## Languages
- [x] Englisj
- [ ] Chinese(Mandarin?)
- [ ] French
- [ ] Spanish
- [ ] Portuguesse

**Comment by WebShivam** [comment]
Hi Agrover! I liked this Project and also loved the Idea for multiple language support
Though I didn't find Hindi  (Indian Language) in the list above I'm willing to translate this to hindi. I'd appreciate if you could assign me this task

**Comment by Agrover112** [comment]
Sure go ahead @WebShivam

**Comment by WebShivam** [comment]
Thank you! It's done now you can look at PR #41

**Comment by Uklizdev** [comment]
Hi! If you assign this to me I can translate it to Spanish 🤓

**Comment by Agrover112** [comment]
Hello @Uklizdev go ahead! Fork the repo and work on it! I will assign you to it .

**Comment by Uklizdev** [comment]
Thanks! You can find it in pull req #46

**Comment by parth-gpt** [comment]
Hi Agrover! I would like to contribute to this project by converting it to French language. Can you please assign it to me?

**Comment by Agrover112** [comment]
> Hi Agrover! I would like to contribute to this project by converting it to French language. Can you please assign it to me?

Hi, Sure you can also take up for Hindi since it's not updated .

**Comment by parth-gpt** [comment]
> > Hi Agrover! I would like to contribute to this project by converting it to French language. Can you please assign it to me?
> 
> Hi, Sure you can also take up for Hindi since it's not updated .

No, I am more comfortable converting it to French than Hindi. So it will be better for me converting it to French.

**Comment by Agrover112** [comment]
> > > Hi Agrover! I would like to contribute to this project by converting it to French language. Can you please assign it to me?
> > 
> > Hi, Sure you can also take up for Hindi since it's not updated .
> 
> No, I am more comfortable converting it to French than Hindi. So it will be better for me converting it to French.

Sounds good. Go ahead fork it , and make sure to update the fork, while submitting the PR.

**Comment by Agrover112** [comment]
@parth-gpt  Are you working on it ? or should I remove you as an asignee?

**Comment by parth-gpt** [comment]
I guess I already made the pull request for the same.

**Comment by WebShivam** [comment]
Hey 👋 @Agrover112 I have updated the Hindi Translation file, you can find it here #55

**Comment by NotTheRightGuy** [comment]
Hey 👋, I can help convert this to Portuguese.

**Comment by Agrover112** [comment]
> Hey 👋, I can help convert this to Portuguese.

Go ahead

**Comment by NotTheRightGuy** [comment]
@Agrover112 I have submitted PR #60. Can you please let me know if anything needs to be changed? Thank you.

**Comment by Devvrat1010** [comment]
@Agrover112 , can you please assign it to me , i'd like to work on French

**Comment by Devvrat1010** [comment]
I would like to work on chinese as well, can you please assign it to me?

**Comment by Devvrat1010** [comment]
@Agrover112 created a pull request for french .
Please review and let me know for any changes.

**Comment by Devvrat1010** [comment]
@Agrover112 created a pull request for Chinese .
Please review and let me know for any changes.

**Comment by Agrover112** [comment]
> @Agrover112 created a pull request for Chinese .
> 
> Please review and let me know for any changes.

Are you using Google Translate? Or any other AI Model? If yes then I would suggest you to get it reviewed by some Native Chinese Speaker before a PR that might contain inconsistencies.
